### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by sualko**
- [x] add documentation to readme
- [x] explain why we forked the original api
- [x] add toc
- [x] add contributor guidelines

Rendered version: https://github.com/littleredbutton/bigbluebutton-api-php/blob/feat-docs/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,272 @@
...
+
+```php
+use BigBlueButton/BigBlueButton;
+
+$bbb = new BigBlueButton($apiUrl, $apiSecret);
+```
+
+If you didn't use composer before, make sure that you include `vendor/autoload.php`.
+
+#### Test if API url and secret are valid
```
Comment (SamuelWei) [inline README.md:86]: That is ok for now, but a new function for connectivity check would be great. Should I create a new issue?

Comment (sualko) [reply README.md:86]: A pr would be better 😉

Comment (joisarjignesh) [reply README.md:86]: i think we need to add api documentation as well for each function so developer can easily find his needs(understand)  and add [apimate](https://mconf.github.io/api-mate/) url for valid  bigbluebuttonserver api calls.

Comment (sualko) [reply README.md:86]: There are plenty of ways to improve the documentation, but can we merge this as a first step?

Comment (Yurujai) [reply README.md:88]: I'm agree with @sualko. The first step can be merge the PR, and then, we can add more improvements.

Comment (joisarjignesh) [reply README.md:88]: @sualko i just suggest to all but  first merge this pr on next time we can improve the documentation.

**Pull request merged**